test/socket.test.ts

```diff
@@ -110,10 +111,12 @@ describe("SocketProxyProvider", () => {
     provider.stop() // We don't need more proxies.
 
     proxy.write("server proxy->client")
-    assert.equal(await getData(fromServerToClient), "server proxy->client")
+    const dataFromServerToClient = await (await getData(fromServerToClient)).toString()
+    expect(dataFromServerToClient).toBe("server proxy->client")
     client.write("client->server proxy")
-    assert.equal(await getData(fromClientToProxy), "client->server proxy")
-    assert.equal(errors, 0)
```

Comment:
Why were these changes required? Is there any advantage to using jest's assertions over builtin `assert`?

Reply:
Not required, more so for consistency (use `expect` + Jest throughout). Also when someone is looking up documentation, they only have to look in one place (jest docs vs jest docs + assert docs).

Reply:
Sounds good.